keymap_drawer/parse/qmk.py

```diff
@@ -50,23 +50,32 @@ def mapped(key: str) -> LayoutKey:
                 key = self._prefix_re.sub("", key)
             return LayoutKey.from_key_spec(self.cfg.qmk_keycode_map.get(key, key.replace("_", " ")))
 
+        def parse_layer(layer: str) -> int:
+            assert self.layer_names is not None
```

Comment:
If I am not wrong, the only way we know layer names is still via user input, i.e. `--layer-names`. If so, I think it would be nice to have an informative error message here, perhaps like below?
```suggestion
            assert self.layer_names is not None, "Please provide `--layer-names` if you use layer defines in keymap.json"
```